ambari-server/src/main/java/org/apache/ambari/server/state/alert/MetricsUri.java

```diff
@@ -29,15 +29,15 @@
 import com.google.gson.annotations.SerializedName;
 
 /**
- * The {@link AlertUri} class is used to represent a complex URI structure where
+ * The {@link MetricsUri} class is used to represent a complex URI structure where
  * there can be both a plaintext and SSL URI. This is used in cases where the
  * alert definition needs a way to expose which URL (http or https) should be
  * used to gather data. Currently, only {@link MetricSource} uses this, but it
  * can be swapped out in other source types where a plain string is used for the
  * URI.
  */
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public class AlertUri {
+public class MetricsUri {
```

Comment:
I don't think it makes sense to change the name of this class to {{MetricsUri}}. This makes alerts which have nothing to do with metrics, like web alerts, look like they use a metric endpoint. If anything, we can abstract this class out to a regular URI.

Reply:
I'm not seeing any Alert related information in this class, only URI-s and http(s) related configuration. Even MetricsUri sounds too specific if we consider what's inside.